### ISSUE TITLE
[iOS] Wrong weekdays translation

### ISSUE THREAD

**Issue by egor-sorokin** [react-native-datetimepicker/datetimepicker#281]
# Bug report
## Summary
Language on a device is "Dutch" and region is "Netherlands". The app is fully translated in Dutch based on language settings of the device. However the weekdays stay in English.

## Environment info
```bash
"react-native": "^0.61.5"
"@react-native-community/datetimepicker": "^3.0.2"
```

Library version: 3.0.2
iOS version: 14.0

## Steps to reproduce

1. Install the datetimepicker library in react-native app
2. Init datetimepicker in the app
```bash
<DateTimePicker
        mode="date"
        locale={'nl-NL'}
        value={new Date()}
      />
```
3. Use device or set an emulator iPhone 11, iOS 14.0
4. Set device language to Dutch (optional set region Netherlands)
5. Run the app
6. Click on datetimepicker
7. Check (short/long) names of weekdays

Describe what you expected to happen:
(short/long) names of weekdays are translated based on the device language

**Comment by daniel** [comment]
Same problem with other locales, the week days are not translated. Also the "Time" label is not translated.

**Comment by christophemenager** [comment]
I have the same issue, did not find a workaround yet

**Comment by eduayrescarneiro** [comment]
Same problem here

**Comment by ahzahri** [comment]
For info, i could fix the issue using this PR : #522 
Suboptimal and temporary solution but my customer was glad that we didn't rollback into the spinners...

**Comment by vonovak** [comment]
closed by https://github.com/react-native-datetimepicker/datetimepicker/pull/540 (v5 release)
